service.php

```diff
@@ -188,7 +188,7 @@ function page_optimize_build_output() {
 		}
 
 		if ( 'text/css' == $mime_type ) {
-			$dirpath = $subdir_path_prefix . dirname( $uri );
+			$dirpath = '/' . ltrim( $subdir_path_prefix . dirname( $uri ), '/' );
```

Comment:
For general use outside WP.com, I think we probably need to root relative URLs using the site URL like:
- site URL is https://example.com/blog/
- `wp-content/resource` URI becomes `/blog/wp-content/resource`

Reply:
ah... I think I'm wrong. `$subdir_path_prefix` is meant to address that concern.